code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -145,6 +145,8 @@
 	tc = 0
 
 /datum/outfit/syndicate/post_equip(mob/living/carbon/human/H)
+	// We don't *REALLY* require the nukiebase be loaded to function, but lets go ahead and kick off loading just in case
+	INVOKE_ASYNC(SSmapping, TYPE_PROC_REF(/datum/controller/subsystem/mapping, lazy_load_template), LAZY_TEMPLATE_KEY_NUKIEBASE)
```

Comment:
Can we move this? It's silly. Put this in the actual gamemode instead, not the goddamn outfit equipping.

Also, the wizard gamemode (does that exist?) or round event? needs a trigger. Same for nukeops round event